intrinsic_cal/src/robocyl_ical_8D.cpp

```diff
@@ -532,12 +643,24 @@ class robocyl_ical_8D
     ROS_ERROR("could not open covariance file %s", covariance_file_name.c_str());
     return (false);
   };  // end computeCovariance()
+  std::string current_image_file (int scene, std::string& filename){
+    std::string present_image_file_path_name;
+    char scene_chars[8];
+    sprintf(scene_chars,"_%03d.jpg",scene);
+    if(filename == ""){ // build file name from image_directory_,
+      present_image_file_path_name  = image_directory_ + std::string("/") +  camera_name_ + std::string(scene_chars);
+    }
+    else{
+      present_image_file_path_name  = image_directory_ + "/" +  filename;
+    }
+    return present_image_file_path_name;
+  }
```

Comment:
add blank line, or short description above this function, at end put a comment to know that its the end of a funciton